core/src/lib.rs

```diff
@@ -68,6 +67,7 @@ impl Raiko {
             Ok((header, _mpt_node)) => {
                 info!("Verifying final state using provider data ...");
                 info!("Final block hash derived successfully. {}", header.hash());
+                debug!("Final block header derived successfully. {header:?}");
```

Comment:
Why both `debug` and `info` here?

Reply:
info for just `hash` and debug for entire `header`

Reply:
I see that, but the entire header is both `info` and `debug` printed as you can see here:
https://github.com/taikoxyz/raiko/blob/1e9987c1beb0fc70042ccfdae3cdc2bc3a6178e5/core/src/lib.rs#L70-L71

Reply:
oh, missed 71... probably just replaced `println!`. will clean up.